autoload/ctrlp/bookmarks.vim

```diff
@@ -20,10 +20,15 @@ function! ctrlp#bookmarks#init() abort
         let l:line_nrs = sort(bm#all_lines(l:file), "bm#compare_lines")
         for l:line_nr in l:line_nrs
             let l:bookmark = bm#get_bookmark_by_line(l:file, l:line_nr)
+            if g:bookmark_show_detail_in_ctrlp
+              let l:detail=fnamemodify(l:file,':t').'|'.l:line_nr.'|'
```

Comment:
Can you please add a space between the line number and the content preview?

Example:
`autoload/bm.vim|8| function! bm#has_bookmarks_in_file(file)`